Dockerfile

```diff
@@ -18,6 +14,14 @@ ENV NOTIFY_BACKUP_SUCCESS false
 ENV NOTIFY_BACKUP_FAILURE false
 ENV BACKUP_TAR_TRIES 5
 ENV BACKUP_TAR_RETRY_SLEEP 30
-
-WORKDIR /dockup
```

Comment:
Any specific reason not to add the workdir again? I would like to keep it compatible to the behavior before.